### DISCUSSION TITLE
How is the inspector working in bit.dev website?

### DISCUSSION THREAD

**Discussion by linonetwo** [teambit/bit#7568] [FAQ] [answered]
We want to implement a similar thing as a debug tool in Tiddlywiki https://github.com/Jermolene/TiddlyWiki5/issues/6345#issuecomment-1603621517 , can you give us some guidance? Thanks!

**Comment by itaymendel** [comment, marked as answer]
this is specifically built for bit-components. you can see it here - https://bit.cloud/teambit/community/ui/community-highlighter
